docs-v2/integrations/all/github.mdx

```diff
@@ -14,32 +14,130 @@ import PreBuiltUseCases from "/snippets/generated/github/PreBuiltUseCases.mdx"
--   [OAuth-related docs](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps)
+| Topic | Links | 
+| - | - | 
+| General | [GitHub Developer Documentation](https://docs.github.com/en/developers) |
+| | [GitHub REST API Documentation](https://docs.github.com/en/rest) |
+| OAuth | [Creating an OAuth App](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/creating-an-oauth-app) |
+| | [Authorizing OAuth Apps](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps) |
+| | [OAuth Scopes for GitHub](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/scopes-for-oauth-apps) |
+| Comparison | [Differences between GitHub Apps and OAuth Apps](https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/differences-between-github-apps-and-oauth-apps) |
+| GitHub Apps | [Registering a GitHub App](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app) |
+| | [GitHub App Permissions](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps) |
 
-<Note>Contribute useful links by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/github.mdx)</Note>
+## Common Scopes
+
+| Scope | Description |
+| ----- | ----------- |
+| `repo` | Full control of private repositories |
+| `repo:status` | Access commit status |
+| `repo_deployment` | Access deployment status |
+| `public_repo` | Access public repositories |
+| `repo:invite` | Access repository invitations |
+| `security_events` | Read security events |
+| `admin:repo_hook` | Full control of repository hooks |
+| `write:repo_hook` | Write repository hooks |
+| `read:repo_hook` | Read repository hooks |
+| `admin:org` | Full control of organizations and teams |
+| `write:org` | Write access to organization |
+| `read:org` | Read access to organization |
+| `admin:public_key` | Full control of user public keys |
+| `write:public_key` | Write user public keys |
+| `read:public_key` | Read user public keys |
+| `admin:org_hook` | Full control of organization hooks |
+| `gist` | Create gists |
+| `notifications` | Access notifications |
+| `user` | Update user information |
+| `read:user` | Read user profile information |
+| `user:email` | Access user email addresses (read-only) |
+| `user:follow` | Follow and unfollow users |
+| `delete_repo` | Delete repositories |
+| `write:discussion` | Write team discussions |
+| `read:discussion` | Read team discussions |
+| `write:packages` | Upload packages to GitHub Package Registry |
+| `read:packages` | Download packages from GitHub Package Registry |
+| `delete:packages` | Delete packages from GitHub Package Registry |
+| `admin:gpg_key` | Full control of user GPG keys |
+| `write:gpg_key` | Write user GPG keys |
+| `read:gpg_key` | Read user GPG keys |
+| `codespace` | Full control of codespaces |
+| `workflow` | Update GitHub Action workflows |
```

Comment:
Instead of providing a list of scopes, specify the link where the scopes are listed on GitHub: https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes